dgbpy/torch_classes.py

```diff
@@ -424,8 +423,9 @@ def __init__(self,
 
         self.cbs = []
         defaultCBS = [ TrainEvalCallback(), AvgStatsCallback(metrics),
-                        EarlyStoppingCallback(earlystopping), TensorBoardLogCallback()]
+                        EarlyStoppingCallback(earlystopping) ]
         if not hasFastprogress(): self.silent = True
```

Comment:
Should this be removed if we will be using a Bokeh progressbar?

Reply:
Yes, all these will slightly change when the bokeh progress bar is added.

Fastprogress is currently the default for handling notebook and terminal progress bar during training.